types/generated/contentTypes.d.ts

```diff
@@ -1516,6 +1570,7 @@ declare module '@strapi/types' {
       'api::save-idea.save-idea': ApiSaveIdeaSaveIdea;
       'api::sendmail.sendmail': ApiSendmailSendmail;
       'api::team-membership.team-membership': ApiTeamMembershipTeamMembership;
+      'api::time-capsule.time-capsule': ApiTimeCapsuleTimeCapsule;
```

Comment:
@RETprojects Could you please remove extra line. We have already removed TimeCapsule